processing/src/main/java/org/apache/druid/query/groupby/epinephelinae/GroupByMergingQueryRunnerV2.java

```diff
@@ -280,12 +280,17 @@ public AggregateResult call()
               return RowBasedGrouperHelper.makeGrouperIterator(
                   grouper,
                   query,
-                  () -> Lists.reverse(resources).forEach(CloseQuietly::close)
+                  resources
               );
             }
             catch (Throwable e) {
               // Exception caught while setting up the iterator; release resources.
-              Lists.reverse(resources).forEach(CloseQuietly::close);
+              try {
+                resources.close();
+              }
+              catch (IOException ex) {
```

Comment:
I don't think it makes sense to catch anything weaker than `Throable` in another `catch (Throwable e) {` block.

Reply:
This can't be weaker since it is being thrown by the `.close()` method: https://github.com/apache/incubator-druid/blob/master/core/src/main/java/org/apache/druid/java/util/common/io/Closer.java#L155

Reply:
Why it can't be weaker? You can catch a weaker Exception.

Reply:
Ah like that, yes that's perfectly possible 👍